web/templates/page/about.html.eex

```diff
@@ -0,0 +1,53 @@
+<p class="b">What is Healthlocker?</p>
+
+<p>Healthlocker is a secure platform powered by the
+  <a href="http://www.slam.nhs.uk/" target="_blank">South London and Maudsley Trust</a>
+  that promotes supported self-management and opportunities to improve
+  communication between service users, carers and clinicians.</p>
+
+<p class="b">It is currently in development and this is the Beta version. More features
+  and improvements will be added in the coming months.</p>
+
+<p class="b">Who is Healthlocker for?</p>
+
+<p>Anyone can <a href="/users/new">sign up to Healthlocker</a> to explore
```

Comment:
Even for these links it'd be best to use the Phoenix helper [`link/2`](https://hexdocs.pm/phoenix_html/Phoenix.HTML.Link.html#link/2).